lua/neogit/buffers/commit_editor/init.lua

```diff
@@ -40,25 +39,24 @@ function M:open()
     modifiable = true,
     readonly = false,
     autocmds = {
-      ["BufWritePre"] = function()
-        written = true
-      end,
       ["BufUnload"] = function(o)
-        if written then
-          if
-            not config.values.disable_commit_confirmation
-            and not input.get_confirmation("Are you sure you want to commit?")
-          then
-            -- Clear the buffer, without filling the register
-            vim.api.nvim_buf_set_lines(o.buf, 0, -1, false, {})
-            vim.api.nvim_buf_call(o.buf, function()
-              vim.cmd("silent w!")
```

Comment:
Why not just `return`? Doing exploratory testing through some scenarios I couldn't find a difference using this vs a simple return.